pkg/operator/ceph/object/spec_test.go

```diff
@@ -379,3 +379,18 @@ func TestCheckRGWKMS(t *testing.T) {
 	assert.True(t, b)
 	assert.NoError(t, err)
 }
+
+func TestGetDaemonName(t *testing.T) {
+	context := &clusterd.Context{Clientset: test.New(t, 3)}
+	store := simpleStore()
+	c := &clusterConfig{
+		context: context,
+		store:   store,
+	}
+	resourceName := fmt.Sprintf("%s-%s", AppName, c.store.Name)
+	rgwConfig := &rgwConfig{
+		ResourceName: resourceName,
+	}
+	daemon := getDaemonName(rgwConfig)
+	assert.Equal(t, "ceph-client.rgw.default", daemon)
```

Comment:
The daemon log also has the daemon ID which is missing in the example, e,g: `ceph-client.rgw.my.store.a.log`. Let's make this example more realistic.

Reply:
I would also suggest that we should have 2 test cases for the unit test to make sure that it works as expected for different inputs.

Reply:
Done